### PR TITLE
Removed required parameter

### PR DESCRIPTION
**Pull request by dorsha**
fixes: https://github.com/demisto/server/issues/8387

### CODE REVIEW

File: Integrations/integration-Guidance_Encase_Endpoint.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ configuration:
   name: port
   defaultvalue: ""
   type: 0
-  required: true
+  required: false
```
Comment (aviadl) [inline Integrations/integration-Guidance_Encase_Endpoint.yml]: So you need a default port if you make it non required

Comment (dorsha) [inline Integrations/integration-Guidance_Encase_Endpoint.yml]: OK, what will be the default port?

Comment (aviadl) [inline Integrations/integration-Guidance_Encase_Endpoint.yml]: I have no idea :) @orius123  ?

Comment (orius123) [inline Integrations/integration-Guidance_Encase_Endpoint.yml]: Maybe 8700. Saw it somewhere as default and it is as good as anything else..

Comment (dorsha) [inline Integrations/integration-Guidance_Encase_Endpoint.yml]: We will leave it required.
We didn't found the original field that was not required.

**Pull request closed without merging**